libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -374,10 +387,19 @@ static void eventCallback( MQTTContext_t * pContext,
     assert( pContext != NULL );
     assert( pPacketInfo != NULL );
 
+    if( pPacketInfo->type == disconnectOnPacketType )
+    {
+        /* Terminate MQTT connection with server for session restoration test. */
+        TEST_ASSERT_EQUAL( MQTTSuccess, MQTT_Disconnect( &context ) );
+
+        /* Graceless disconnect. Terminate TLS session and TCP connection. */
```

Comment:
Don't think it's graceless since we already sent the disconnect packet.
```suggestion
        /* Graceful disconnect. Terminate TLS session and TCP connection. */
```

Reply:
This comment still needs to be fixed

Reply:
Fixed to avoid use of _graceless_